### PR TITLE
Command processing, Username Highlighting, Bumped Version

### PR DESCRIPTION
**Pull request by Bottswana**
Added filtering for madden bots (using settings.channel prefix to bypass peoples filtering)
Added options for trivia. Enable and whitelist trivia bot username. Adds colouring to stand out from chat.
Removed forced entry of settings.channel into input box and replaced with intelligent prefixing.
- A - in front of message will send message without prefixing your settings.channel (for trivia for example)
- The  / commands are now ignored for prefixing and will run correctly
- settings.channel prefix being typed by the user will not prepend another.

### CODE REVIEW

**Comment by Bottswana** [comment]
Missed latest changes, just going to merge those in.

File: robin.user.js [hunk 1]
```diff
@@ -158,6 +158,8 @@
     Settings.addInput("spamFilters", "Custom spam filters, comma delimited.", "spam example 1, spam example 2");
     Settings.addInput("channel", "Channel filter", "");
     Settings.addBool("filterChannel", "Filter by channel", false);
+    Settings.addBool("showtrivia", "Show/Highlight Trivia", false);
+    Settings.addInput("triviahosts", "List of Trivia Hosts", "dthunder,nbagf");
```
Comment (dashed) [inline robin.user.js]: Trivia is too niched. Just rename this to a **username highlighter**.

**Comment by dashed** [comment]
@Bottswana you may want to change the title of this issue

**Comment by Bottswana** [comment]
There you go haha

**Comment by dashed** [comment]
LGTM. /cc @vartan @devoidfury

**Comment by vartan** [comment]
I think the trivia is basically a duplicated feature of user muting. I think users can mute trivia themselves if we get the mute patched up correctly. I think it is supposed to save now but my local copy isn't saving anything for some reason, it may be a bug

**Comment by Bottswana** [comment]
It's highlighting, not muting, so no I disagree that it is a duplicate of
muting, since the feature is exactly the opposite.

It allows you to highlight a person/bot of interest that you would like to
see more prominent than the rest of the chat.

**Comment by vartan** [comment]
OH i missed that, I apologize. I will say then, the madden filtering is a duplicate of "custom spam filters, comma delimited" then, but I think I added that after this PR was generated.

Could you rebase and remove the madden stuff? I'll merge it in! Great features.

**Comment by Bottswana** [comment]
Sure I can do that, no worries.

**Comment by Bottswana** [comment]
@vartan Done. Also bumped version number.

**Pull request merged**